src/imputation/expansion_imputation.py

```diff
@@ -18,38 +18,46 @@ def evaluate_imputed_ixx(
     """Evaluate the imputed 2xx or 3xx as the sum of all 2xx or 3xx
     over the sum of all 211 or 305 values, multiplied by the imputed 211.
     """
+    imp_class = group["imp_class"].values[0]
+    ExpansionLogger.debug(f"Imputation class: {imp_class}.")
+    ExpansionLogger.debug(f"Master column: {master_col}.")
+
+    # Return the groupby object unaltered if there are no TMI values
+    # in the imputation class
+    TMI_mask = group["imp_marker"] == "TMI"
```

Comment:
I think I'm just missing this, but where do we set the values in the `imp_marker column`?

Reply:
in imputation_main

Reply:
and in the `apply_tmi` func in tmi_imputation sets the TMI value
                imp_class_df["imp_marker"] = "TMI"